authentication/models.py

```diff
@@ -0,0 +1,66 @@
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+from django.db import models
+
+
+# Setting Up Custom User
+
+# Creating a custom user manager
+class UserManager(BaseUserManager):
```

Comment:
Let's move the Managers to a separate file for modularity.

Reply:
done